cpp/template/cmake/thirdparty/fetch_rapids.cmake

```diff
@@ -0,0 +1,20 @@
+# =============================================================================
+# Copyright (c) 2023, NVIDIA CORPORATION.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except
+# in compliance with the License. You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software distributed under the License
+# is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+# or implied. See the License for the specific language governing permissions and limitations under
+# the License.
+
+set(RAPIDS_VERSION "23.04")
```

Comment:
I would add a comment that this version needs to match the RAFT version in get_raft.cmake for people unfamiliar with our versioning

Reply:
Ah- yes it is already using the RAPIDS_VERSION from here. I've added a comment to the code.